Source/Cotter.swift

```diff
@@ -38,7 +38,18 @@ public class Cotter {
         // Set the resource bundle
         let frameworkBundle = Bundle(for: Cotter.self)
         let bundleURL = frameworkBundle.resourceURL?.appendingPathComponent("Cotter.bundle")
+        let bundlePath = frameworkBundle.resourcePath
+        print(frameworkBundle.resourcePath ?? "no path")
         return Bundle(url: bundleURL!)!
+//        guard let bundleURL = frameworkBundle.url(forResource: "Cotter", withExtension: "bundle") else {
```

Comment:
I think we should remove these commented lines unless you're planning to keep it for something